### ISSUE TITLE
SQLite - Unable to find the requested .Net Framework Data Provider

### ISSUE THREAD

**Issue by statler** [sjh37/efreversepoco#809]
I am having trouble generating the SQLite configs when using an EF7 library. I am getting the following error

System.ArgumentException: 'Unable to find the requested .Net Framework Data Provider.  It may not be installed.'

Happens at:
`var factory = DbProviderFactories.GetFactory(providerName);`

I know this is caused by the factory not being registered - which is understandable in a framework app, but this is not how the EF Core config works - e.g. there is no app.config. Further, the provider is Microsoft.data.sqlite.

I expect there is something hacky that has to be done to get the factopry to work. Can you please advise - maybe some docs or something?

**Comment by sjh37** [comment]
Yes you are correct you need to register it. Have a read of the [Sqlite wiki here](https://github.com/sjh37/EntityFramework-Reverse-POCO-Code-First-Generator/wiki/SQLite) hopefully that will get you up and running in no time.

**Comment by sjh37** [comment]
## Already Solved

This is a known requirement — the SQLite provider must be registered so the T4 template engine can find it. Please see the [SQLite wiki page](https://github.com/sjh37/EntityFramework-Reverse-POCO-Code-First-Generator/wiki/SQLite) for step-by-step instructions. Closing as this is documented.
